code/LeftAndMain.php

```diff
@@ -1347,7 +1311,7 @@ public function getNewItem($id, $setID = true)
         return $object;
     }
 
-    public function delete($data, $form)
+    public function delete(array $data, Form $form): HTTPResponse
```

Comment:
We cannot, however I think it's fine to change this here because it seems pretty like a fairly unlikely thing to get swapped out in custom code.  The only response negotiator used in admin is `PjaxResponseNegotiator`

If if it has been swapped out, this error should show up pretty quickly on CMS 5 upgrade during admin smoke testing.  Realistically the only thing other than an HTTPReponse returned here seems like it would be a string, and it's easy enough to just `return HTTPResponse::create()->setBody($mystring);`